build_all/base-libraries/AzureIoTUtility/src/samd/time.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#if defined(ARDUINO_ARCH_SAMD)
+#if defined(ARDUINO_ARCH_SAMD) && !defined(WIO_TERMINAL)
```

Comment:
This code is not needed on the Wio Terminal, these functions are already implemented in the Seeed libraries.